src/Api/Management/Entity/App.php

```diff
@@ -187,6 +187,9 @@ public function setCallbackUrl(string $callbackUrl): void
      */
     public function getCredentials(): array
     {
+        usort($this->credentials, function (AppCredentialInterface $a, AppCredentialInterface $b) {
```

Comment:
```suggestion
        usort($this->credentials, static function (AppCredentialInterface $a, AppCredentialInterface $b) {
```

as a minor memory usage improvement

Reply:
Also, consider using arrow functions for typing less :)